include/stdlib.h

```diff
@@ -1,7 +1,7 @@
 #ifndef _STDLIB_H
-# define _STDLIB_H 1
+#define _STDLIB_H 1
```

Comment:
I think that @d0p1s4m4 like those spaces in her coding style (not sure) :thinking:

Reply:
I'm simply following the clang-format